c7n/filters/core.py

```diff
@@ -476,6 +478,10 @@ def process_value_type(self, sentinel, value, resource):
         if self.vtype == 'normalize' and isinstance(value, six.string_types):
             return sentinel, value.strip().lower()
 
+        elif self.vtype == 'base64':
```

Comment:
can probably drop this, i think i had it as experiment to see if it would work for userdata decoding, but its not particularly useful as there multiple forms of userdata possible.

Reply:
Tests pass without it, so I removed.